### DISCUSSION TITLE
[Feature] Federation with other immich instances

### DISCUSSION THREAD

**Discussion by Blintmester** [immich-app/immich#13146] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'm a happy user of immich and some of my friends are hosting there own instances as well as me. 
I've been thinking about that it would be super nice if I could share my album from immich.instance1.com to a user on immich.instance2.com. 

I use (as my users) immich instead of Google Photos, and it's nice, but I'm missing the feature to share (almost) everybody my photos, albums, but with the current implementation I'm only able to share the album(s) to the internal users, or I can share an external link, but I can't share to other users from other immich.

Thanks for considering this feature, it would be super nice!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#728
